<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Attaques</class>
 <widget class="QWidget" name="Attaques">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Attaques</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(18, 18, 18);
color:white</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="gridLayoutMenuAttaques">
     <item>
      <widget class="QLabel" name="labelLogo">
       <property name="minimumSize">
        <size>
         <width>65</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>42</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../resources/images/logo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAccueilAttaques">
       <property name="font">
        <font>
         <family>JetBrainsMono Nerd Font</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #00C853; /* Vert */
    color: white;
    font-weight: bold;
    border-radius: 5px;
    padding: 10px;
}
QPushButton:hover {
    background-color: #009624; / survol */
}</string>
       </property>
       <property name="text">
        <string>Accueil</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonActualiteAttaques">
       <property name="font">
        <font>
         <family>JetBrainsMono Nerd Font</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #00C853; /* Vert */
    color: white;
    font-weight: bold;
    border-radius: 5px;
    padding: 10px;
}
QPushButton:hover {
    background-color: #009624; / survol */
}</string>
       </property>
       <property name="text">
        <string>Actualit√©</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDocumentationAttaques">
       <property name="font">
        <font>
         <family>JetBrainsMono Nerd Font</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #00C853; /* Vert */
    color: white;
    font-weight: bold;
    border-radius: 5px;
    padding: 10px;
}
QPushButton:hover {
    background-color: #009624; / survol */
}</string>
       </property>
       <property name="text">
        <string>Documentation</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDeconnexionAttaques">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../resources/images/deconnexion.png</normaloff>../resources/images/deconnexion.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayoutContentAttaques">
     <item row="6" column="0">
      <widget class="QCheckBox" name="checkBoxHEADERSCOOKIES">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>JetBrainsMono Nerd Font</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox {
    color: white;
}

QCheckBox::indicator {
    width: 25px;
    height: 25px;
    border-radius: 8px;
    border: 2px solid #00FF00; 
    background: black;
}

QCheckBox::indicator:checked {
    background: #00FF00; 
}</string>
       </property>
       <property name="text">
        <string>Headers &amp; Cookies</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="pushButtonLancerAttaques">
       <property name="font">
        <font>
         <family>JetBrainsMono Nerd Font</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #00C853; /* Vert */
    color: white;
    font-weight: bold;
    border-radius: 10px;
    padding: 10px;
}
QPushButton:hover {
    background-color: #009624; / survol */
}</string>
       </property>
       <property name="text">
        <string>Lancer </string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="checkBoxCSRF">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>JetBrainsMono Nerd Font</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox {
    color: white;
}

QCheckBox::indicator {
    width: 25px;
    height: 25px;
    border-radius: 8px;
    border: 2px solid #00FF00; 
    background: black;
}

QCheckBox::indicator:checked {
    background: #00FF00; 
}</string>
       </property>
       <property name="text">
        <string>CSRF</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="10" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelChoixAttaques">
       <property name="font">
        <font>
         <family>JetBrainsMono Nerd Font</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Choisissez votre attaque:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="checkBoxSQLI">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>JetBrainsMono Nerd Font</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox {
    color: white;
}

QCheckBox::indicator {
    width: 25px;
    height: 25px;
    border-radius: 8px;
    border: 2px solid #00FF00; 
    background: black;
}

QCheckBox::indicator:checked {
    background: #00FF00; 
}</string>
       </property>
       <property name="text">
        <string>SQLI</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelNomAttaques">
       <property name="font">
        <font>
         <family>JetBrainsMono Nerd Font</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Attaques</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="checkBoxXSS">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>JetBrainsMonoNL Nerd Font</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox {
    color: white;
}

QCheckBox::indicator {
    width: 25px;
    height: 25px;
    border-radius: 8px;
    border: 2px solid #00FF00; 
    background: black;
}

QCheckBox::indicator:checked {
    background: #00FF00; 
}</string>
       </property>
       <property name="text">
        <string>XSS</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
